docs/modules/ROOT/pages/network-files.adoc

```diff
@@ -0,0 +1,64 @@
+= Network Files
+
+OpenZeppelin Upgrades keep track of all the implementation contracts you have deployed in an `.openzeppelin` folder in the project root, as well as the proxy admin. You will find one file per network there. It is advised that you commit to source control the files for all networks except the development ones (you may see them as `.openzeppelin/unknown-*.json`).
+
+> Note: the format of the files within the `.openzeppelin` folder is not compatible with those of the xref:cli::index.adoc[OpenZeppelin CLI]. If you want to use these plugins for an existing OpenZeppelin CLI project, we will be sharing soon a guide on how to migrate.
+
+
+[[network.json]]
+== `<network_name>.json`
+
+OpenZeppelin Upgrades will generate a file for each of the networks you work on (`ropsten`, `mainnet`, etc). These files share the same structure:
+
+[source,json]
+----
+// .openzeppelin/<network_name>.json
+{
+  "manifestVersion": "3.0",
+  "impls": {
+    "...": {
+      "address": "...",
+      "txHash": "...",
+      "layout": {
+        "storage": [...],
+        "types": {...}
+      }
+    },
+    "...": {
+      "address": "...",
+      "txHash": "...",
+      "layout": {
+        "storage": [...],
+        "types": {...}
+      }
+    }
+  },
+  "admin": {
+    "address": "...",
+    "txHash": "..."
+  }
+}
+----
+
+For every logic contract, besides the deployment address, the following info is also tracked:
+
+* `types` keeps track of all the types used in the contract or its ancestors, from basic types like `uint256` to custom `struct` types
+* `storage` tracks the storage layout of the linearized contract, referencing the types defined in the `types` section, and is used for verifying that any storage layout changes between subsequent versions are compatible
+
+The naming of the file will be `<network_name>.json`, but note that `<network_name>` is not taken from the name of the network's entry in the Truffle or Buidler configuration file, but is instead inferred from the canonical network id associated to the entry. 
+
+[[configuration-files-in-version-control]]
+== Configuration Files in Version Control
+
+Public network files like `mainnet.json` or `ropsten.json` should be tracked in version control. These contain valuable information about your project's status in the corresponding network; the addresses of the contract implementations that have been deployed, the addresses of the proxies that have been deployed, etc. Such files should be identical for all the contributors of a project.
+
+However, local network files like `unknown-<network_id>.json` only represent a project's deployment in a temporary local network such as `ganache-cli` that are only relevant to a single contributor of the project and should not be tracked in version control.
```

Comment:
```suggestion
However, local network files like `unknown-<chain_id>.json` only represent a project's deployment in a temporary local network such as `ganache-cli` that are only relevant to a single contributor of the project and should not be tracked in version control.
```

Reply:
This is generally true but the "unknown" files could also correspond to other public chains that we're unaware of, like the Ethereum Classic chain. I think we should explain that.

Reply:
Added explanation for limited set of public chains
https://github.com/OpenZeppelin/openzeppelin-upgrades/pull/142/commits/3c56793eafe3fd699e4cb6629af098df824f6684